text/0000-standard-dependency-metadata-format.md

```diff
@@ -0,0 +1,48 @@
+# Standardized Paketo Dependency Metadata Format
+
+## Summary
+
+This proposal suggests that the Paketo project should converge on a single dependency metadata format.
+
+## Motivation
+
+There are several projects, such as the introduction of ARM or the removal of stacks, that are being discussed that would force Paketo to update the metadata for dependencies. It seems prudent that if we are going to have to do a large update to dependency metadata that it might also be a good time for us to converge as a project on the dependency metadata format that we should be using going forward.
+
+This could also be a good first step in converging on shared tooling. By having a shared dependency metadata format we would have a good common convergence point to begin building universal tooling for the Paketo project. 
+
+## Implementation
+
+The following in the proposed metadata format:
+
+```toml
+[[metadata.dependencies]]
+arch = "<dependency compatible architecture>"
+checksum = "<dependency algo:checksum>"
+id = "<dependency ID>"
+os = "<dependency compatible OS>"
+uri = "<dependency URI>"
+version = "dependency version"
+
+cpes = [ "<dependency cpe>" ] #optional
+name = "<dependency name>" #optional
+purl = "<dependency purl>" #optional
+strip-components = <number of directories to strip off dependency artifact> #optional
+
+    [[metadata.dependencies.distros]] #optional
+    name = "<compatible OS distribution name>"
+    version = "<compatible OS distribution version>" #optional
+
+    [[metadata.dependencies.licenses]] #optional
```

Comment:
I did a quick [comparison](https://gist.github.com/modulo11/1e8a5ef74f6408ddba40c209521a8765) which showed that the `licenses` field is of different type between libpak and packit. While the approach chosen in the RFC would be also my prefered way, it leads to additional (one-time) work on maintainers of packit based buildpacks.

Reply:
Yes, I think that it will require a little bit of work for packit but I think it is ultimately a relatively small amount of pain.